styles.js

```diff
@@ -15,4 +15,9 @@ export default createGlobalStyle`
   body {
     font-family: ${nunito.style.fontFamily}, system-ui;
   }
+
+input, select, button {
+  font-family: ${nunito.style.fontFamily}, system-ui;
```

Comment:
I am assuming you want the font-family to always match the general font-family set in the body, if that is the case, then there is an easier way to achieve this:

```suggestion
  font-family: inherit;
```

If you changed your body's font-family, you input, select and button elements would automatically use the right font and you would not have to change it in this location as well.

Reply:
Thanks, that's a good tip.